rfcs/20210129-optimize-pd-creation/README.md

```diff
@@ -0,0 +1,262 @@
+# Optimization of Primitive Descriptor Creation
+
+## Motivation
+
+There are three majors steps to create a oneDNN primitive:
+1. Create operation descriptor
+2. Create primitive descriptor
+3. Create primitive
+
+In frameworks, all the three steps are performed every time when a user executes
+an operation. The last two steps are expensive hence they can significantly
+affect performance of the operation.
+
+oneDNN recently introduced a cache for primitives to significantly reduce primitive
+creation time - the most expensive step. The second most expensive step is creating
+the primitive descriptor. The cost may very from several microsecond to hundreds
+of microseconds depending on number of the primitive implementations and time of
+primitive descriptor initialization.
+Such overhead is significant because the execution time can also fall in this range.
+This is the motivation to start optimizing the primitive descriptor creation step.
```

Comment:
I believe this sentence is redundant, the entire paragraph is about the motivation.

Reply:
I don't mind removing it if you think it's redundant.